snippets/capabilities-table.adoc

```diff
@@ -28,6 +28,9 @@ The following table describes the `baselineCapabilitySet` values.
 |`v4.16`
 |Specify this option when you want to enable the default capabilities for {product-title} 4.16. By specifying `v4.16`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.16 are `baremetal`, `MachineAPI`, `marketplace`, `OperatorLifecycleManager`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot`, `NodeTuning`, `ImageRegistry`, `Build`, `CloudCredential`, `DeploymentConfig`, and `CloudControllerManager`.
 
+|`v4.17`
```

Comment:
The dev-branch/4.18 will want `v4.18` too (openshift/api#2022) but the 4.17 backport only needs `v4.17` (openshift/api#2023).  I dunno how you want to structure that in docs.  Could be either:

1. Land this pull with just `v4.17` in dev/4.18.
2. Robot cherry-pick back to 4.17.
3. Follow-up dev/4.18 pull to add `v4.18`.

or:

1. Land a dev/4.18 pull adding both `v4.18` and `v4.17`.
2. Manual backport to 4.17 that only adds `v4.17`.

I don't have an opinion on which will be less work for docs.

Reply:
I'll just open it again on enterprise-4.17 :) Thanks!